partcad/src/partcad/part_config.py

```
@@ -49,9 +50,12 @@ def normalize(name, config):
                         "default": param_value,
                     }
                 # All params are float unless another type is explicitly speciifed
-                elif (
-                    isinstance(param_value, dict) and not "type" in param_value
-                ):
+                elif isinstance(param_value, dict) and not "type" in param_value:
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)